### PR TITLE
Add meta charset

### PR DESCRIPTION
**Pull request by muffinresearch**
Add `<meta charset="utf-8">`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta charset="utf-8">
```
Comment (wagnerand) [inline index.html]: Shouldn't this tag be a singleton? https://www.w3.org/International/questions/qa-html-encoding-declarations.en

Comment (muffinresearch) [inline index.html]: Do you mean self-closing? If so specifying the trailing '/' is optional and ignored in HTML5 for void elements. [1]

[1] https://www.w3.org/TR/html5/syntax.html#void-elements

Comment (wagnerand) [inline index.html]: TIL, thanks!

**Comment by muffinresearch** [comment]
@wagnerand thanks for the r? Can you land it I don't have the commit bit?

**Pull request merged**